web3c/utils.js

```diff
@@ -1,12 +1,13 @@
 /**
  * objectAssign is an alternative to Object.assign to work with react-native.
- * Assigns all properties from b onto a.
+ * Assigns all properties from b onto a as readonly by default.
```

Comment:
Why readonly as the default?  If you used `Object.assign` the properties wold be writable.